pages/dkp/kommander/2.1/networking/load-balancing/index.md

```diff
@@ -59,14 +59,26 @@ If the virtual IP addresses share an interface with the primary IP address of th
 
 MetalLB can be configured in two modes: Layer2 and BGP.
 
-The following example illustrates how to enable MetalLB and configure it with the Layer2 mode:
+The following example illustrates how to enable MetalLB and configure it with the Layer2 mode.
+
+First, set the `WORKSPACE_NAMESPACE` environment variable using the following command to get the name of the workspace's namespace that you would like to deploy MetalLB to.
+MetalLB will be deployed to all attached clusters within this workspace.
+
+```bash
+export WORKSPACE_NAMESPACE=$(kubectl get workspace <type_your_workspace_name> -o jsonpath='{.status.namespaceRef.name}')
+```
+
+<p class="message--note"><strong>NOTE:</strong> To deploy MetalLB to the Kommander host cluster, this will be the `kommander` namespace.</p>
```

Comment:
```suggestion
<p class="message--note"><strong>NOTE:</strong> To deploy MetalLB to the Kommander host cluster, this will be the <code>kommander</code> namespace.</p>
```